packages/flutter/lib/src/material/tab_controller.dart

```diff
@@ -83,12 +83,38 @@ class TabController extends ChangeNotifier {
       assert(initialIndex != null && initialIndex >= 0 && (length == 0 || initialIndex < length)),
       _index = initialIndex,
       _previousIndex = initialIndex,
-      _animationController = length < 2 ? null : AnimationController(
+      _animationController = AnimationController.unbounded(
         value: initialIndex.toDouble(),
-        upperBound: (length - 1).toDouble(),
         vsync: vsync,
       );
 
+  // Private constructor used by `copyWith`. This allows a new TabController to
+  // be created without having to create a new animationController.
+  TabController._({
+    int index,
+    int previousIndex,
+    AnimationController animationController,
+    @required this.length,
+  }) : _index = index, _previousIndex = previousIndex, _animationController = animationController;
+
+
+  /// Creates a new [TabController] with `index` and `length` if they are
```

Comment:
Since this is now private, it should use // instead of ///

Reply:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-for-public-quality-private-documentation

Reply:
OK, I'm going to leave as `///` especially since I can imagine us one day making this method public.